core/store/src/lib.rs

```diff
@@ -222,12 +222,13 @@ impl NodeStorage {
     /// store, the view client should use the split store and the cold store
     /// loop should use cold store.
     pub fn get_split_store(&self) -> Option<Store> {
-        match &self.cold_storage {
-            Some(cold_storage) => Some(Store {
-                storage: crate::db::SplitDB::new(self.hot_storage.clone(), cold_storage.clone()),
-            }),
-            None => None,
-        }
+        self.get_split_db().map(|split_db| Store { storage: split_db })
+    }
+
+    pub fn get_split_db(&self) -> Option<Arc<SplitDB>> {
```

Comment:
this is refactor in order to be able to get SplitDB explicitly. Without this, we only use if behind a Store interface.